### ISSUE TITLE
[BUG] Atom vs Atomic problem marked as solved in #4536 still a problem

### ISSUE THREAD

**Issue by NatashaMartinez** [deepmodeling/deepmd-kit#4601]
### Bug summary

Training atomic dipoles returns nan values unless the data is labelled as atom_*.npy instead of atomic_*.npy. 

### DeePMD-kit Version

DeePMD-kit v3.0.1

### Backend and its version

TF

### How did you download the software?

conda

### Input Files, Running Commands, Error Log, etc.

{
  "_comment": "that's all",
  "model"
: {
    "type_map"
: [
     "C",
     "N",
     "O",
     "H",
     "X",
     "Z"
    ],
    "descriptor": {
      "type" : "se_e2_a",
      "sel": "auto",
      "rcut_smth": 0.5,
      "rcut" : 6.0,
      "neuron": [
       16,
       32,
       48],
      "resnet_dt": false,
      "seed": 1,
      "_comment": " that's all"    },
    "fitting_net": {
       "type": "dipole",
       "sel_type": [2],
       "neuron"
: [
       150,
       150,
       150
      ],
      "resnet_dt": true,
      "seed": 1222,
      "activation_function": "tanh",
      "_comment": " that's all"
    },
    "_comment": " that's all"
  },
  "learning_rate"
: {
    "type": "exp",
    "decay_steps": 1000,
    "start_lr": 5e-4,
    "stop_lr": 1e-8,
    "_comment": "that's all"
  },
  "loss"
: {
    "type": "tensor",
    "pref": 0.0,
    "pref_atomic": 1.0,
    "_comment": " that's all"
  },
  "training"
: {
    "training_data"
: {
      "systems"
: [
       "./training_data"
      ],
      "batch_size": "auto",
      "_comment": "that's all"
    },
    "validation_data":{
          "systems": ["./validation_data"],
                "batch_size": "auto",
      "_comment": "that's all"
    },
    
    "numb_steps": 325000,
    "seed":7515,
    "disp_file": "lcurve.out",
    "disp_freq": 100,
    "save_freq": 100,
    "_comment": "that's all"
  },

  "_comment": "that's all"
}

### Steps to Reproduce

Training atomic dipole model from wannier data

**Comment by anyangml** [comment]
The fix provided by #4538 is after the release of v3.0.1. Can you try the `devel` branch?

**Comment by NatashaMartinez** [comment]
Indeed, I missed that it is a part of the 3.0.2 milestone. Thanks for clarifying.
